#PLANNING CONFIGURATIONS:
group_name: "cartesian_arm"
utopia_tolerance: 0.05
extend: false
max_distance: 0.5
start_configuration: [0.0,0.0,0.0]
stop_configuration:  [3.0,3.0,3.0]
solver_time: 60.0

# HUMAN-AWARE COST FUNCTION CONFIGURATION
base_frame: "world"
tool_frame: "end_effector"
max_step_size: 0.05
object_type: "red_box"
n_object: 10
n_threads: 10
